botstrap.py

```diff
@@ -83,11 +83,37 @@ def __init__(self, guild_id: int | str):
             event_hooks={"response": [self._raise_for_status]},
         )
         self.guild_id = guild_id
+        self._app_info: dict[str, Any] | None = None
 
     @staticmethod
     def _raise_for_status(response: Response) -> None:
         response.raise_for_status()
 
+    @property
+    def app_info(self) -> dict[str, Any]:
+        """Fetches the application's information."""
+        if self._app_info is None:
+            response = self.get("/applications/@me")
+            self._app_info = cast("dict[str, Any]", response.json())
+        return self._app_info
+
+    def upgrade_application_flags_if_necessary(self) -> bool:
+        """
+        Set the app's flags to allow the intents that we need.
+
+        Returns a boolean defining whether changes were made.
+        """
+        # Fetch first to modify, not overwrite
+        current_flags = self.app_info.get("flags", 0)
+        new_flags = current_flags | 1 << 15 | 1 << 19
```

Comment:
Those bit shifts are also "magic numbers", aren't they ? it'd better to assign each one to a variable indicating the flag.